testhelper/httptest.go

```diff
@@ -0,0 +1,81 @@
+// Package testhelper implements utility routines required for writing unit tests.
+// The testhelper should only be used in unit tests.
+package testhelper
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/cloudflare/cfssl/crl"
+	"golang.org/x/crypto/ocsp"
+)
+
+type LogWriter struct{ *log.Logger }
+
+func (w LogWriter) Write(b []byte) (int, error) {
+	w.Printf("%s", b)
+	return len(b), nil
+}
+
+func MockServer(cert *x509.Certificate, key *rsa.PrivateKey, desiredOCSPStatus ocsp.ResponseStatus, revokedInCRL bool, wg *sync.WaitGroup) *http.Server {
```

Comment:
package [httptest](https://pkg.go.dev/net/http/httptest#NewServer) should be used as a mock server instead of a real server.